deepmd/entrypoints/train.py

```diff
@@ -216,6 +218,90 @@ def train(
         _do_work(jdata, run_opt)
 
 
+def train_mt(
+    *,
+    INPUT: str,
+    init_model: Optional[str],
+    restart: Optional[str],
+    output: str,
+    mpi_log: str,
+    log_level: int,
+    log_path: Optional[str],
+    **kwargs,
+):
```

Comment:
Basically `train_mt` is copied from `train`, this is NOT recommended. 
We do not want to maintain two copies of codes.